apps/mobile-wallet/package.json

```diff
@@ -31,6 +31,7 @@
     "@react-navigation/elements": "^1.3.14",
     "@react-navigation/material-top-tabs": "^6.6.2",
     "@react-navigation/native": "^6.1.3",
+    "@react-navigation/native-stack": "^6.11.0",
```

Comment:
The screen transactions are really not smooth anymore on Android...

## In this PR 

https://github.com/user-attachments/assets/3021b51e-0a23-47ec-8c82-7ff046821e44

## In banxa-next



https://github.com/user-attachments/assets/108d43f2-f498-4c9a-88ed-4b60aeb06e18

Reply:
Fixed in follow up PR!